no-computer.md

```diff
@@ -10,41 +10,82 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+My non-computer DSL idea is actually an abstract game I am in the process of 
+creating for my analog game design class. The game is called ["Circle of Paranoia"](https://docs.google.com/document/d/1WssK6n2gQf6TpReBtXNDcGkmX7z2NeDntuUo4jmU3aI/edit?usp=sharing), 
+and the game meets the need of people who are bored and want to play an amusing 
+game that requires some spatial thinking! It is a very accessible game that young
+kids could play using luck, but for adults, it could be a very competitive strategy
+game. Many abstract strategy games require a lot of time, but this game is pretty 
+short and fun to play in your spare time. 
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+In this game, I would consider the different colored pieces as the vocabulary
+of the DSL, and the different positions on the board as the governing syntax. 
```

Comment:
I would also consider each move a syntax, and look somewhat like this:

if(this.left.occupied(true)){
this.swap(this.left);
}